README.md

```diff
@@ -114,10 +149,68 @@ In case of problems with the build, please tell us including the following thing
 - CMakeCache.txt
 - If you are using windows, please include the output of ```cygcheck -s ```
 
-## Source Directory Structure
+
+# Basic Usage
+
+Once compiled, the VM entry point is at `./build/dist/pharo`.
+Since PharoVM is only a VM, an image is required to do useful things.
+
+An image used for the build process is available at `./build/vmmaker/image/Pharo*.image` (glob here because the name is variable since it contains version number and hash).
+
+```bash
+$ ./build/dist/pharo ./build/vmmaker/image/Pharo*.image eval '3+4'
+7
+```
+
+Or run the graphical environment with.
+
+```bash
+$ ./build/dist/pharo ./build/vmmaker/image/Pharo*.image --interactive
+```
+
+Note: since this image is used by the build process, it is not advised to alter it, but you can still save it as a copy.
+
+## Hacking
+
+### Editing the VM code in your image
```

Comment:
This section looks a bit like a duplicate of https://github.com/pharo-project/pharo-vm/wiki/Simulation-Environment

Reply:
This subsection is not original and comes from aaa9c4b2228f879b97bdffae0b0d78557f7caeca (Jan 2020). I did only some minimal change.

The wikipages need some love anyway.